### ISSUE TITLE
French translations - TODO’s and open Q’s

### ISSUE THREAD

**Issue by lucsurr** [fetlife/translations#46]
This issue is intended to gather French translations’ todo’s and open questions.

I) `tu` or `vous`
============

We have to choose between 'tu' and 'vous' (i.e. between informal/familiar, and formal/polite way to address to the user)! Mixing both is really odd…

II) Choice of word(s)
=================

 -  'amour' for `loves` sounds odd in french… maybe 'cœur' (like 'coup de cœur')? 
 - 'lignes directrices' for `guidelines`… works, but maybe 'charte' would be better?
 - how to translate `(activity) feed`? 'Fil (d’activités)'?
 - `legalese`: 'mentions légales' is the canonical terminology, but a bit formal compared to `legalese`, could also use 'Blabla légal'? Or, more pedant, 'Logorrhée légale'?

**Comment by johnbaku** [comment]
@lucsurr I always use vous when speaking French... it's just the way I was brought up... didn't have any other choice! :-p BUT... everyone at work would make funny of me when I would address them as vous. So yeah, I don't know.

**Comment by rakvat** [comment]
@johnbaku in German I used "Du" instead of "Sie" which would be "tu" in French.

**Comment by lucsurr** [comment]
Yep, tbh I would not mind either, just want to have one for sure, and update whole translation to use it :tongue: 

Within a more “traditional” context I would use “vous” without hesitation - but FetLife has a bit of an offbeat tone, so “tu” might fit better in this case… :wink:

**Comment by yianni3000** [comment]
hey @lucsurr ! 1. Regarding tu or vous, I have always preferred vous. I understand it is formal, but I always liked that approach. It is just a layer of respect between two people. I could be super old fashioned. 2. For guidelines, i liked lignes directrices. But of course, maybe i am wrong :/ 3.  For legalese, I love love your suggestion of Blabla légal. Funny! 4. The "love" button was a tricky one. I decided with aimer as a verb. You are right, amour is off, but it is closer related to aimer. Maybe coeur could work as the "love counter" i.e 18 coeurs. 5. For activity feed, I had no solid idea as to what was correct :/

**Comment by reneklacan** [comment]
For Slovak translations I chose less conservative and more friendly addressing from those two options. But it every language there's different threshold...

**Comment by Wapdoowap** [comment]
As a pro translator, I'm often confronted with the "tu or vous" issue. Usually "vous" is preferred, and used much more often when other languages like German, Dutch (or Spanish I think) would use a more familiar "tu" form. I just changed the entire file with "vous" forms.

**Comment by Wapdoowap** [comment]
## **Nickname**: what are we going to use for this? To me "**Pseudo**" would work, shorter, or "**Nom d'utilisateur**" but it's longer and more formal.

**Comment by lucsurr** [comment]
Yeah, re tu/vous, also inclined to stick to “vous”, so let’s bust our “tu” usages :tongue: 
@yianni3000 agree with your points, “activity feed” will go as “fil d’activité” then I guess…
@Wapdoowap For nickname, would also go with “pseudo”, short and commonly used.

@Wapdoowap you are working on french translation right now, right?

**Comment by johnbaku** [comment]
@lucsurr said:

> Within a more “traditional” context I would use “vous” without hesitation - but FetLife has a bit of an offbeat tone, so “tu” might fit better in this case… 😉

And it makes me want to use "tu" for FetLife even if I use vous when I talk. But if you guys already have vous in place we can get a feel for vous and if it feels off and un-fetlife then we can switch to tu.

**Comment by Wapdoowap** [comment]
@lucsurr I am! I went through everything quickly once, but once we're decided on certain words, etc, I can certainly reread everything.

@johnbaku, honestly, I think using "tu" here would feel off. Websites tend to do that when they address children and teenagers, but not much past that. I understand wanting to come off as friendly, but I don't think the "tu" would work at all here...

**Comment by yianni3000** [comment]
@lucsurr  @Wapdoowap  @johnbaku  Vous it is!

**Comment by johnbaku** [comment]
If everyone is good with "vous" I am good with vous!

**Comment by erickgirard** [comment]
@Wapdoowap Not quite sure that "pseudo" would fit here.

The context where "nom d'utilisateur" is used is more as in "login name", "pseudo" would be more used for a "screen name" type of context.

In this case both are interchangable since it's the same thing for now. Maybe "Identifiant" could fit better considering you don't usually log yourself with a screen name ?

**Comment by erickgirard** [comment]
I am also questioning the use of "mail" in french considering that generally the accept term is either "courriel" or "adresse courriel".

On a more general field, what "kind" of french should it be targeted to ? France or International or somewhere in between ? (I know some terms in France's french are generally not accepted as part of the french usage for us Québécois 😄)

**Comment by lucsurr** [comment]
Yeah… I tried to mix a bit between “mail” (very common in France’s french) and “courriel” (which is quite used in Quebec afaik, and is the official choice of the academy, but not that commonly used in real life here). Same with “spam” vs. “pourriel”. I’d say we can keep mixing both, can help avoiding repetitions (which is always a good thing), and think any french speaker will understand both.

Regarding “pseudo” vs. “identifiant”, don’t have strong opinion, both work (though the former is a bit more friendly, casual, than the later), we can always change that later if it becomes necessary anyway.

As for which ”french” to target… I’d say France one, which is supposed to be the reference - but I guess not everyone will agree on that last statement, and I’m obviously biased :wink:

**Comment by Wapdoowap** [comment]
@erickgirard @lucsurr - ooh, point. I'm Franco-Belgian and I don't know nuthin' about them Quebecois uses. 
I went for straight French-from-France uses, but indeed there are some things we should decide on together - like, a **phone text message** is "texto" in France, "SMS" in Belgium. No idea about Quebec. 

Same for **e-mail**. Here "courriel" (being indeed the official recommendation from the Academy) sounds very official/administrative/formal; almost everyone routinely says "mail". I guess we could use "e-mail" or "mail" (not as in the text now, "email", spelled like "enamel"). But, Quebec? I don't know what is commonly used.

What do we do then?

Re: **avoiding repetitions**: I agree it's a good thing to do in a long text, but I'm afraid it would be counter-productive in text that is essentially a users' guide. In this case I beleive consistency is essential, why risk making people wonder if we're talking about the same thing?

**Comment by Wapdoowap** [comment]
Also, **technical question**: I apologize for the newbie wondering here, but I've never used Github before and I'm not sure I get how it works. Before I found the comment / discussion section here, I went to [https://github.com/fetlife/translations/blob/master/locales/fr.yml](https://github.com/fetlife/translations/blob/master/locales/fr.yml) , corrected what I thought needed it, and it opened a "branch". Same when I went in later. But I'm not finding my two branches anymore. I see that you all have USERNAME/translations branches. Is someone collecting and integrating the branching translations? It would be silly for me to redo the work y'all already did...

**Comment by Wapdoowap** [comment]
Yet another point: 
## inclusive language.

I believe it's very important to use language that includes the feminine form, too, and not address every user as if they were just all dudes. ;)

We all agree, right?

In my branch I used the classic "_(e)_", feminine forms in ( ).
There are now other spellings to include the feminine, with dashes or middle-dots (like "_motivé-e-s_" or "_motivé·e·s_") but they have a "militant feminist" vibe, and don't feel laid-back at all. 

Not that _I_ personally mind that, myself, but do we want to give that vibe?

**Comment by lucsurr** [comment]
Re feminine (or plural) forms, yes, they are totally needed. I tend to prefer `motivé.e` over `motivé(e)`, I find it a bit more inclusive, parenthesis can have a feeling of “oh, btw, yes, we also include feminine gender here”…

Anyway, there’s no way to convey all the other genders (a-, bi-, fluid, other, etc.) either, French is a horrible bi-gendered language (which historically always used masculine forms for any kind of neutral/generic addressing)! :grin:

**Comment by lucsurr** [comment]
@Wapdoowap you should read https://guides.github.com/ but basically, once you have saved your work in own branch of own repo, you have to submit a **pull request** to fetlife's repo to get it reviewed and merged, you should have a green button for that on your branch's page… And let’s try to avoid further tech issue like that in this ticket, it’s already quite populated by translations discussions ;)

**Comment by Wapdoowap** [comment]
@lucsurr - I agree that the "(e)" feel like an afterthought. Which is why the alternative spellings were developed. 
Thing is, those alternatives do feel very "militant" - which is why I try to use them in my private writings, but not in my pro translations. It's adding an "actively feminist" tone that my clients didn't intend in the first place ;)

 My only wonder is if we, and our esteemed Great Leader here, want Fet to give off that vibe? ;)

**Comment by markusb** [comment]
I just came across this repository because the French version of Fetlife is broken at the moment.

There is a discussion about it here: https://fetlife.com/groups/693/group_posts/10315423

Essentially the translation is only displayed on a very limited set of pages. /languages /about /legalese /guidelines do work. /home /groups /events /places do not work.

I have also tried other languages, the only one working is German, all others are Broken.

Am I the only one ?

**Comment by reneklacan** [comment]
@markusb see FAQ in the README

**Comment by markusb** [comment]
I think you refer to the only FAQ here:

    Q. Why are only some pages in FetLife translated? We are only translating pages that have been redesigned and made mobile friendly. As we move pages to the new design we make them translatable.

The FAQ item is nice, but useless I do not not know which pages are redesigned and which are not. From German working on the main feed I presume translation should work there and therefore French too, which is doesn't.

**Comment by Shell4000** [comment]
### French translation

Greetings, 

Over time, I have watched this site struggle with translations (French) even being answered by the caretaker that "No, we don't need your help, we got it covered" that was a while ago and since nothing more got translated. 

The following translation may not be 100% but it's sure not far from it and those 3 translations may have taken me about 6-8 hrs top to produce...I have been doing this for over 30yrs. 
Sorry, I do not know how to use the codes needed for this site nor do I have the time or wish to educate myself in that regard.

I hope this can be of use and if not, all is not lost as I did enjoy working on it and if you are interested, I also have a list of the main fetishes translated give or take a few that I didn't stop to dwell on but if needed for the cause, it would be a simple revision on my part and final draft on yours like the following. 

If you ever need any more translations, let me know.

Btw, the French groups from "France" in my personal opinion are not interested any more than needed in having the following translation available.
What they cannot read and understand, they cannot apply, follow or be accused of...

In regards to using 
**"Tu** ou **Vous"** 
In general, **Tu** will apply to someone very familiar or your buddy (example)
**Vous** is the only proper acceptable way especially with the French from Europe and when translating an international site like this one, the translation will mostly favour them not the French "québequois" Canadian but again, as translation is by the book, The Larousse French Dictionary is Master. 

[Traduction des Lignes Directrices Fetlife relatives au "Contenu".](https://fetlife.com/users/4784573/posts/6430725)

[Traduction des Lignes directrices Communautaires Fetlife](https://fetlife.com/users/4784573/posts/6430695)

[Traduction des Lignes directrices FetLife - Les Valeurs](https://fetlife.com/users/4784573/posts/6430675)

Best Regards.
